examples/index.json

```diff
@@ -4619,6 +4619,34 @@
       }
     ]
   },
+  {
+    "title": "YAML data",
+    "column": 3,
```

Comment:
This column may be too long now.

Also see https://github.com/ballerina-platform/ballerina-distribution/issues/5537

Reply:
No need to fix others, but can we at least make sure this is added okay?

Reply:
updated the Table to column 2

Reply:
Missed that this was being added under common libraries. If so, the table change isn't relevant?

Btw we have xmldata and jsondata examples under Language concepts.

Reply:
Ping on this.